tests/all/wasi_testsuite.rs

```diff
@@ -38,37 +104,55 @@ fn is_wasm(entry: &DirEntry) -> bool {
     path.is_file() && (ext == Some("wat") || ext == Some("wasm"))
 }
 
-fn run<P: AsRef<Path>>(module: P) -> Result<Output> {
-    run_wasmtime_for_output(
-        &[
-            "run",
-            "--wasi-modules",
-            "experimental-wasi-threads",
-            "--wasm-features",
-            "threads",
-            "--disable-cache",
-            module.as_ref().to_str().unwrap(),
-        ],
-        None,
-    )
+fn should_ignore<P: AsRef<Path>>(path: P, ignore_list: &[&str]) -> bool {
+    let file_name = path.as_ref().file_name().unwrap().to_str().unwrap();
+    ignore_list.contains(&file_name)
 }
 
-fn parse_spec<P: AsRef<Path>>(spec_file: P) -> Result<Spec> {
-    let contents = fs::read_to_string(spec_file)?;
-    let spec = serde_json::from_str(&contents)?;
-    Ok(spec)
+fn build_command<P: AsRef<Path>>(module: P, extra_flags: &[&str], spec: &Spec) -> Result<Command> {
+    let mut cmd = get_wasmtime_command()?;
```

Comment:
Is it worth pulling this call outside the loop in `run_all` and passing the result in? It looks like the result should be the same across iterations of that loop.

Reply:
Oh, I think that might make sense. We would still have to clone the `Command` because we mutate it but that is not a big deal.

Reply:
Hm, there is no `Command::clone` (learn something new every day!) so I propose we don't do this.